lib/coinbase.rb

```diff
@@ -1,11 +1,14 @@
 # frozen_string_literal: true
 
+module Coinbase; end
+
+require_relative 'coinbase/client'
```

Comment:
The client module needs to be loaded first because we're catching the `ApiError` and handling it based off of the error code.

Reply:
Looks like `ApiError` is not the underlying type that we defined but rather a generic error, so this does not provide the actual server-provided error code.